gradle.properties

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-ideaVersion = IC-15.0.6
+ideaVersion = IU-162.1447.7
```

Comment:
@aslo - is the specific build number (162.14...) needed or can we achieve the same thing with 2016.2 here? If so that would make it clearer.